src/components/material/MaterialDescription.tsx

```diff
@@ -87,7 +87,7 @@ const MaterialDescription: React.FC<MaterialDescriptionProps> = ({ work }) => {
                 dataCy="material-description-series-members"
               />
             )}
-            {subjectsList && (
+            {subjectsList.length > 0 && (
```

Comment:
This is the third time we have to implement this check. Have you considered moving the length check into the `HorizontalTermLine` component?